### PR TITLE
58: Add Drupal Console. 59: Fix composer install

### LINKED ISSUE #58: Install drupal console

### PR DESCRIPTION
**Pull request by kporras07**
Fixes #58 and #59

### CODE REVIEW

File: generators/app/templates/8/docs/_build.md [hunk 1]
```diff
@@ -13,5 +13,5 @@ Builds are created in the `web` directory.
 ## Executing a build
 
 ```bash
-npm run build-site
```
Comment (rigoucr) [inline generators/app/templates/8/docs/_build.md:16]: What means this change?

Comment (kporras07) [reply generators/app/templates/8/docs/_build.md:16]: I also included composer install cleanup into this PR. So, it means to remove that command in favor of simply `composer install`

**Comment by rigoucr** [comment]
@kporras07 I'm getting this error on vagrant up
drupalconsoleerror

**Comment by kporras07** [comment]
@rigoucr did you generate a new project to test this? Are you sure generator_mdsk code was up-to-date in your env?

**Comment by rigoucr** [comment]
yes

**Comment by rigoucr** [comment]
sorry my mistake :(. :shipit:

**Pull request merged**